frontend/pages/docs/users/v-two-docs/creating-a-model.mdx

```diff
@@ -0,0 +1,53 @@
+import Image from 'next/legacy/image'
+import Box from '@mui/material/Box'
+import bailoMarketIcon from '../../../../public/docs/bailo_market_icon.png'
+
+## **Bailo Marketplace**
+
+The models added to Bailo are displayed in the Bailo Marketplace.
+
+<Box sx={{ maxWidth: '75%', margin: 'auto' }}>
+  <Image src={bailoMarketIcon} />
+</Box>
+
+This enables the models to be discoverable to other users. There's an exception to this, where the models are marked as
+private. In these instances the model will not be displayed.
+
+## **Finding models**
+
+Models are discoverable by the search functionality or via filtering on the tags.
+
+## **Adding a model to the Marketplace**
+
+Selecting to **_Add a new model_** will enable you to create a model.
```

Comment:
"New models can be added to Bailo using the the 'Add a new Model' button on the Marketplace, or alternatively, the 'Add Model' button at the top of the application."